features/graph/documentation/README.md

```diff
@@ -0,0 +1,36 @@
+# Netlify Graph
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Components](#components)
+- [Enabling the feature](#enabling-the-feature)
+- [Feedback](#feedback)
+
+## Overview
+
+Netlify Graph is a new feature released through [Netlify Labs](https://www.netlify.com/blog/2021/03/31/test-drive-netlify-beta-features-with-netlify-labs/), that enables developers to seamlessly integrate third-party APIs in their web applications without having to write API-specific code.
+
+We handle the messy integration work, and you can focus on solving interesting problems. Instead of connecting different APIs and SDKs with brittle code, you can now abstract all that out behind a convenient GraphQL interface.
+
+## Components
```

Comment:
I'm not sure how people will read these docs, but if we think they'll hop around between the main readme and the component doc pages, we may want to add this "Components" section to every page. This would accessible links to the other doc pages and people wouldn't have to go back to the main readme or navigate the directory tree in GitHub.